src/image-tags.js

```diff
@@ -1,13 +1,29 @@
 import {get, isEmpty} from 'lodash';
 import {FocusedImage} from 'quintype-js';
 
-function pickImageFromCard(story, cardId) {
-  const { metadata = {} } = story.cards.find(card => card.id === cardId) || {};
-  if(metadata && !isEmpty(metadata) && get(metadata, ['social-share', 'image', 'key'], false)) {
-    return new FocusedImage(metadata['social-share'].image.key, metadata['social-share'].image.metadata || {});
+function pickImageFromCard(story, urlQuery = {}) {
+  const {cardId, elementId} = urlQuery;
+  if (!cardId && !elementId) {
+    return undefined;
+  }
+
+  const card = story.cards.find(card => card.id === cardId) || {};
+  const element = elementId && (card['story-elements'] || []).find( element => element.id === elementId);
+  if(element && element['image-s3-key']){
```

Comment:
there is a check if both cardId and elementId is not present. If Card Id is present and no element id it will go through the entire array of story-elements

Reply:
👍